tests/TransliteratorTest.php

```diff
@@ -46,6 +46,7 @@ public function provideTransliterationCases()
             array('това е тестово заглавие', 'tova-e-testovo-zaglavie'),
             array('это тестовый заголовок', 'eto-testovyi-zagolovok'),
             array('führen Aktivitäten Haglöfs', 'fuhren-aktivitaten-haglofs'),
+            array('that it\'s \'eleven\' \'o\'clock\'', 'that-its-eleven-oclock'),
```

Comment:
Ah, did this on Friday, but forgot to push in my rush to flee the office. Pushing now.